rita/src/rita_client/dashboard/neighbors.rs

```diff
@@ -123,17 +140,20 @@ fn generate_neighbors_list(
             // we check that this is safe above
             let exit_route = maybe_exit_route.unwrap();
 
-            output.push(NodeInfo {
-                nickname: nickname.to_string(),
-                ip: identity.mesh_ip.to_string(),
-                id: *identity,
-                route_metric_to_exit: exit_route.metric,
-                route_metric: neigh_route.metric,
-                total_payments: debt_info.total_payment_received.clone(),
-                debt: debt_info.debt.clone(),
-                link_cost: exit_route.refmetric,
-                price_to_exit: exit_route.price,
-            })
+            if let Some(stats_entry) = stats.get(&neigh_route.iface) {
```

Comment:
Right now you won't print an entry at all for a peer if you don't get a stats entry for it. You can either add an else statement or do a nice trick on line 123

let tup = (current_exit, stats.get(&neigh_route.iface))
if let (Some(current_exit), Some(stats_entry) = tup) {
...
}
else {
 ....
}

What this does is let you reuse the else state by pattern matching on a tuple, in the cases where one is some and the other is None it will go to else. Anyway it's slightly cleaner with less code duplication really a nit.